### PR TITLE
Week_02

### PR DESCRIPTION
**Pull request by AndreLinhares-Dev**

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    padding: 10px;
+}
+
+main{
+    max-width: 840px;
+}
+aside{
+    width: 12rem;
+    margin: 2rem;
+    border: 1px solid rgba (0,0,0, 0.2);
```
Comment (chrsmendes) [inline styles/styles.css]: ### Error Message
[CSS Validation Service](https://jigsaw.w3.org/css-validator/validator?uri=AndreLinhares-Dev.github.io/wdd130/)
image

### Solution
The CSS Validation Service is returning an error in your CSS. To resolve this, you simply need to remove the space between the `rgb` function and its parameters. It should look like this: `rgb(0, 0, 0, 0.2);`

File: week02/excersise2/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bos21001) [inline week02/excersise2/index.html:6]: ### Error Message
image

### Solution
Add a meta description in the `<head>` tag, like this: `<meta name="description" content="[the description here]">`.

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    padding: 1rem;
+    background-color: #eee;
+    color: #333;
+    text-align: center;
+}
```
Comment (chrsmendes) [inline styles/styles.css]: ### Error Message
image

### Solution
Refer back to the course content to resolve this issue: [Course Content](https://byui-cse.github.io/wdd130-ww-course/week02/prove-home-enhancement-css.html#:~:text=Set%20the%20img%20inside%20the%20aside%20to%20have%20a%20width%20of%20200%20pixels%20and%20a%20height%20of%20auto).

Comment (chrsmendes) [inline styles/styles.css]: ### Error Message
image

### Solution
Refer back to the course content to resolve this issue: [Course Content](https://byui-cse.github.io/wdd130-ww-course/week02/prove-home-enhancement-css.html#:~:text=Change%20the%20existing%20paragraph%20p%20padding%20property%20value%20to%200%20pixels.%0ANote%20that%20any%200%20value%20does%20not%20need%20to%20specify%20the%20units).

Comment (chrsmendes) [inline styles/styles.css]: ### Error Message
image

### Solution
Refer back to the course content to resolve this issue: [Course Content](https://byui-cse.github.io/wdd130-ww-course/week02/prove-home-enhancement-css.html#:~:text=Apply%20CSS%20to%20the%20%3Cfooter%3E%20element%20to%201%EF%B8%8F%E2%83%A3add%20a%20top%20margin%20to%20separate%20it%20from%20the%20main%20element%20content%2C%202%EF%B8%8F%E2%83%A3add%20a%20top%20border%20to%20design%20additional%20distinction%20from%20the%20main%20content%2C%20and%203%EF%B8%8F%E2%83%A3align%20the%20any%20text%20in%20the%20footer%20to%20the%20center%20of%20its%20content%20box.).

**Review by chrsmendes — CHANGES_REQUESTED** [review]
Hey Andre, could you please review the comments I added to your pull request and address them as soon as possible?

File: index.html [hunk 4]
```diff
@@ -4,6 +4,9 @@
 
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta name="Homepage" content="Homepage do Andre">
+    <meta name="author" content="Andre Linhares de Souza">
+    <link rel="stylesheet" href="styles/styles.css">
```
Comment (chrsmendes) [inline index.html:9]: ### Error Message
image

### Solution
Add a meta description in the `<head>` tag, like this: `<meta name="description" content="[the description here]">`.

**Review by chrsmendes — APPROVED** [review]
Great job!!

**Comment by chrsmendes** [comment]
Well done!

**Pull request merged**